lib/src/folding/parsers/spaces_foldable_block_parser.dart

```diff
@@ -0,0 +1,109 @@
+import 'dart:collection';
+
+import 'package:charcode/ascii.dart';
```

Comment:
Package imports and relative imports should be separated by a blank line.
https://dart.dev/guides/language/effective-dart/style#ordering